### PR TITLE
SLING-5725 Removed dependency on sling-testing-tools for testing http clients

### PR DESCRIPTION
**Pull request by dulvac**
- adapted BundlesInstaller
- Removed WebconsoleClient
- Added methods to manipulate bundles to OsgiConsoleClient

### CODE REVIEW

File: testing/http/clients/src/main/java/org/apache/sling/testing/clients/osgi/OsgiConsoleClient.java [hunk 1]
```diff
@@ -276,16 +296,279 @@ public String editConfigurationWithWait(int waitCount, String PID, String factor
...
+     * Install a bundle using the Felix webconsole HTTP interface, with a specific start level
+     * @param f
+     * @param startBundle
+     * @param startLevel
+     * @return the sling response
+     * @throws ClientException
+     */
+    public SlingHttpResponse installBundle(File f, boolean startBundle, int startLevel) throws ClientException {
+        // Setup request for Felix Webconsole bundle install
+        FormEntityBuilder builder = FormEntityBuilder.create();
```
Comment (kwin) [inline testing/http/clients/src/main/java/org/apache/sling/testing/clients/osgi/OsgiConsoleClient.java]: I don't see the file being added to the POST request

File: testing/http/clients/src/main/java/org/apache/sling/testing/clients/osgi/OsgiConsoleClient.java [hunk 2]
```diff
@@ -27,12 +30,19 @@
...
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.net.URI;
 import java.util.*;
+import java.util.jar.JarInputStream;
+import java.util.jar.Manifest;
 
 import static org.apache.http.HttpStatus.SC_MOVED_TEMPORARILY;
 import static org.apache.http.HttpStatus.SC_OK;
```
Comment (kwin) [inline testing/http/clients/src/main/java/org/apache/sling/testing/clients/osgi/OsgiConsoleClient.java:49]: Adding a hint in the classes javadoc on http://felix.apache.org/documentation/subprojects/apache-felix-web-console/web-console-restful-api.html would be nice.

**Comment by dulvac** [comment]
Fixed the problem in installBundle + added javadoc reference to the webconsole http api

**Comment by rombert** [comment]
With the upcoming migration to git this repository will be deleted. If the pull request contains data that should be re-submitted I suggest that you save the changes locally and resubmit them after the migration, which will likely happen on Monday the 9th of October.

**Pull request closed without merging**